README.md

```diff
@@ -113,17 +113,65 @@ Write a conditional statement that...
 ### Your solution here:
 8.  Write your javascript solution below
 ```js
-// your answer here
+
+var number = i
+var result=" ";
+for(var i=0; i <=100; i++){
+
+if(number % 3==0){
```

Comment:
You missed this condition `- If it is a multiple of both 3 and 5, print “FizzBuzz” instead of the number.`